chapter05/jxp/ch5ex2.pl

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+use feature "say";
+
+say "Please enter some lines of text";
+my @lines = <STDIN>;
+
+for (my $i = 1; $i <= 30; $i++){
+  print substr( $i, -1 );
+}
+print "\n";
+
+# One extra for the newline character
+foreach my $line ( @lines ){
```

Comment:
you can use the concatenation operator to avoid this loop e.g. printf "%21s" x @lines, @lines; (scalar then list context)